normalize.css

```diff
@@ -0,0 +1,427 @@
+/*! normalize.css v6.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
rather than checking in this file into your version control you could:

A. install the `normalize-css` NPM package
B. link to normalize via a CDN
C. install normalize via `bower`

all of those options are preferable to checking in vendor code.